_emails/us/north_carolina/wilmington.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Wilmington, NC
+permalink: "/wilmington-nc"
```

Comment:
Why not just
permalink: "/wilmington"

?

Reply:
because wilmington, delaware already exists.